controllers/dspipeline_params.go

```diff
@@ -584,6 +585,7 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 	p.PiplinesCABundleMountPath = config.GetCABundleFileMountPath()
 	p.PodToPodTLS = false
 	p.FIPSEnabled = config.GetBoolConfigWithDefault(config.FIPSEnabledConfigName, config.DefaultFIPSEnabled)
+	p.WebhookName = "kubernetes-webhook"
```

Comment:
It'd be good to make this a constant somewhere because other parts of the code hardcode this name.